src/DIRAC/Core/DISET/private/Service.py

```diff
@@ -241,11 +243,10 @@ def _initMonitoring(self):
         props = [("__doc__", "description")]
         for prop in props:
             try:
-                value = getattr(self._handler["module"], prop[0])
+                getattr(self._handler["module"], prop[0])
             except Exception as e:
                 gLogger.exception(e)
                 gLogger.error("Missing property", prop[0])
```

Comment:
Are these 7 lines needed at all?

Reply:
I agree it is probably not something that needs this kind of runtime checks. Removing it.